scripts/dockerfiles/Dockerfile.release

```diff
@@ -20,6 +20,9 @@ COPY ["LICENSE", "NOTICE", "/"]
 
 COPY out/amazon-ecs-pause.tar /images/amazon-ecs-pause.tar
 
+# Copy our cni plugins ecs-eni, ecs-ipam and ecs-bridge
+COPY out/cni-plugins /cni-plugins
```

Comment:
Hmm, so if customers want to bring their own CNI plugins, they are expected to perform two things:
* Mount cni-plugin directory explicitly to a non-conflicting path
* Set the config `ECS_CNI_PLUGINS_PATH` accordingly

I'll go with your recommendation here but we should document the process of adding a custom CNI plugin to the agent in the README